src/Evolution/Executables/GrMhd/GhValenciaDivClean/GhValenciaDivCleanBase.hpp

```diff
@@ -635,8 +636,9 @@ struct GhValenciaDivCleanTemplateBase<
                                           non_tensor_compute_tags>,
                 Events::time_events<system>,
                 control_system::control_system_events<control_systems>,
-                intrp::Events::Interpolate<3, InterpolationTargetTags,
-                                           interpolator_source_vars>...>>>,
+                intrp::Events::InterpolateWithoutInterpComponent<
+                    volume_dim, InterpolationTargetTags,
+                    interpolator_source_vars>...>>>,
```

Comment:
I think you can remove at least the `intrp::Interpolator` component below in the `component_list`, maybe more things (@knelli2?). Looks like it can be removed in `EvolveValenciaDivClean` as well (maybe also in other executables, haven't checked).

Reply:
It can't be removed in `GhValencia` because the "with horizon" executables need it for horizon finding (maybe we could add another template bool to include it or not). It could probably be removed from `Valencia` because I don't think it makes sense to find a horizon in a static spacetime.

Regardless, let's do those in a followup so @isaaclegred can actually get waveforms from his BNS runs now.